buildSrc/src/main/java/org/elasticsearch/gradle/precommit/ThirdPartyAuditTask.java

```diff
@@ -171,19 +172,30 @@ private void extractJars(FileCollection jars) {
         File jarExpandDir = getJarExpandDir();
         // We need to clean up to make sure old dependencies don't linger
         getProject().delete(jarExpandDir);
-        jars.forEach(jar ->
+
+        jars.forEach(jar -> {
```

Comment:
This seems to get us the merged/latest version of each jar only? This seems like an ok start in order to handle MRJARs, but I think we should at least have a note that the limitation on checking exists here.